examples/RetrieveAuditEvents.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016, 2018, Oracle and/or its affiliates. All rights reserved.
+'''
```

Comment:
Favour block comments (one `#` per line) as per PEP8: https://www.python.org/dev/peps/pep-0008/#block-comments rather than the triple quote syntax